resources/public/api/conf/3.0/common/errors.yaml

```
@@ -303,3 +303,10 @@ components:
         code: RULE_ALREADY_OPTED_OUT
         message: Customer is already opted out of coding out
 
+    #400
+    formatChargeReference:
+      description: |
+        The format of the supplied charge reference field is not valid.
```

It is not a field (it's a parameter):
```suggestion
        The format of the supplied charge reference is not valid.
```
May need to update spec.

---

Looks good. Still need to update for v2.0 errors.yaml.